db/models/product.js

```diff
@@ -1,33 +1,36 @@
 'use strict'
 
 const Sequelize = require('sequelize')
+const {STRING, TEXT, INTEGER} = require('sequelize')
 
 module.exports = db => db.define('product', {
   title: {
-    type: Sequelize.STRING,
+    type: STRING,
     allowNull: false
   },
 
-  description : Sequelize.TEXT,
+  description : TEXT,
 
   price: {
-    type: Sequelize.INTEGER,
+    type: INTEGER,
     DefaultValue: 0.00
```

Comment:
defaultValue -> d is lowercase 
also if this is an integer, the default value should be 0 w/o decimals